resources/jest-preprocessor.js

```diff
@@ -1,12 +1,17 @@
+/* tslint:disable */
+
 const tsc = require('typescript')
 const tsConfig = require('../tsconfig.json')
 
-const { compilerOptions } = tsConfig
+const compilerOptions = tsConfig.compilerOptions
 
 module.exports = {
   process (src, fileName) {
     // Make sure that source maps are always enabled and inline so that they
     // can be picked up by the `source-map-support` package.
-    return tsc.transpileModule(src, { compilerOptions, fileName }).outputText
+    return tsc.transpileModule(src, {
+      compilerOptions: compilerOptions,
+      fileName: fileName,
+    }).outputText
```

Comment:
Yep. That got me at first, but this file does not run through a build step so we can’t use new syntax if we want to run CI for Node.js v4.